db/schema.rb

```diff
@@ -25,11 +28,11 @@
     t.index ["author_id"], name: "index_author_identities_on_author_id"
   end
 
-  create_table "authors", force: :cascade do |t|
-    t.integer "cap_profile_id", limit: 4
+  create_table "authors", id: :serial, force: :cascade do |t|
+    t.integer "cap_profile_id"
```

Comment:
You'll notice a lot of missing `limit: 4` on integer fields, and that's because a limit of `4` is the default setting for PG.